Manifest.toml

```diff
@@ -659,9 +660,11 @@ version = "1.2.0"
 
 [[deps.Oceananigans]]
 deps = ["AMGX", "Adapt", "AlgebraicMultigrid", "CUDA", "CUDAKernels", "Crayons", "CubedSphere", "Dates", "DocStringExtensions", "FFTW", "Glob", "IncompleteLU", "InteractiveUtils", "IterativeSolvers", "JLD2", "KernelAbstractions", "LinearAlgebra", "Logging", "MPI", "NCDatasets", "OffsetArrays", "OrderedCollections", "PencilArrays", "PencilFFTs", "Pkg", "Printf", "Random", "Rotations", "SeawaterPolynomials", "SparseArrays", "Statistics", "StructArrays", "Tullio"]
-git-tree-sha1 = "569005a95a472facb8e0bc35d4c3b29308eb82e7"
+git-tree-sha1 = "d832a1b10421f24cd52c4d4c27cea3f8b1e4b49e"
+repo-rev = "glw/catke-plus-gm"
+repo-url = "https://github.com/CliMA/Oceananigans.jl.git"
```

Comment:
let's wait for the registered Oceananigans version before merging

Reply:
this is just manifest tho right? does it matter?

Reply:
well... this points to a branch which might be deleted at some point
at least we should point to a commit on `#main`?

but I was also a bit confused because [this commit](https://github.com/CliMA/Oceananigans.jl/pull/2844/commits/37c60f934ce284103d56fcd106ae36ea5fee262d) was bumping up a patch release but 0.78.3 already existed... did you mean to do 0.78.4?

Reply:
Also somehow CUDAKernels 0.4 is here but I remember @simone-silvestri saying that this was giving trouble...

Reply:
oh did i screw that up

Reply:
what i mean by "just the manifest" is just that it doesn't affect child packages, it only affects peopel trying to use this manifest (so its not a severe issue) and we can update it soon

Reply:
I thought I checked oceananigans main and saw 0.78.2 and therefore concluded that 0.78.3 was next

Reply:
Then whoever depends directly on CUDAKernels (Oceananigans) should add compat for CUDAKernels; we don't have to fiddle iwth that here

Reply:
OK, agreed... Doesn't matter.

There were two PRs in Oceananigans that bumped the version simultaneously? So https://github.com/CliMA/Oceananigans.jl/pull/2844 is not included in 0.78.3 since it was tagged before it was merged.

Reply:
*sigh* yeah that's what happened.

Reply:
Let's push to 0.78.4